run-checks

```diff
@@ -12,26 +12,10 @@ STATIC_GO=""
 STATIC_JS=""
 UNIT_GO=""
 UNIT_JS=""
-SKIP_NPM_INSTALL="no"
+export PHANTOMJS_BIN=`pwd`/node_modules/.bin/phantomjs
```

Comment:
no sure I understand what you said below, ... I was not talking about $0 there (on the contrary) ... and `pwd` is *very* context specific